File: 2756-buy-two-chocolates/buy-two-chocolates.py
class Solution:
    def buyChoco(self, prices: List[int], money: int) -> int:
        prices.sort()
        cho=0
        temp=money
        for i in prices:
            if i <= temp and temp > -1 and cho < 2:
                cho+=1
                temp -= i
            if cho == 2 and temp > -1:
                return temp
        return money

        

File: 2756-buy-two-chocolates/buy-two-chocolates.java
class Solution {
    public int buyChoco(int[] prices, int money) {
        int cho = 0, temp = money;

        Arrays.sort(prices);
        for (int i=0 ;i < prices.length;i++){
            if (prices[i] <= temp && temp > -1 && cho < 2 ){
                cho++;
                temp-=prices[i];  
            }
            if(cho == 2 && temp > -1){
                return temp;
            }
        }
        return money;
        
    }
}